lib/Elastica/IndexTemplate.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace Elastica;
+
+use Elastica\Exception\InvalidException;
+
+/**
+ * Elastica index template object.
+ *
+ * @author Dmitry Balabka <dmitry.balabka@gmail.com>
+ */
+class IndexTemplate
+{
+    /**
+     * Index template name.
+     *
+     * @var string Index pattern
+     */
+    protected $name = '';
```

Comment:
I just made it like that because in contributing notes is mentioned that code must be in PSR-2 code style, but exactly PSR-2 says that:
> Property names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#42-properties